tests/ui/perfherder/alerts_test.jsx

```diff
@@ -227,6 +244,32 @@ const mockUpdateAlertSummary = (alertSummaryId, params) => ({
   failureStatus: null,
 });
 
+const alertsView = () =>
+  render(
+    <AlertsView
+      user={testUser}
+      projects={repos}
+      location={{
+        pathname: '/alerts',
+        search: '',
+      }}
+      history={createMemoryHistory('/alerts')}
+      frameworks={[
+        { id: -1, name: 'all' },
+        { id: 1, name: 'talos' },
+        { id: 2, name: 'build_metrics' },
+        { id: 4, name: 'awsy' },
+        { id: 5, name: 'awfy' },
+        { id: 6, name: 'platform_microbench' },
+        { id: 10, name: 'raptor' },
+        { id: 11, name: 'js-bench' },
+        { id: 12, name: 'devtools' },
+        { id: 13, name: 'browsertime' },
+        { id: 14, name: 'vcs' },
```

Comment:
These names are duplicated in `options` on line 210, so mapping over this array to get your options would be dry-er.

Reply:
Done.